data/records/totem-datasets-2011.json

```diff
@@ -0,0 +1,195 @@
+        "type": "double",
+        "unit": "Milimeters",
+        "variable": "vtx_x_L"
+      },
+      {
+        "description": "Milimeters; Reconstructed interaction point vertex x position based on information from right station",
+        "type": "double",
+        "unit": "Milimeters",
+        "variable": "vtx_x_R"
+      },
+      {
+        "description": "Reconstructed interaction point vertex x position",
+        "type": "double",
+        "unit": "Milimeters",
+        "variable": "vtx_x"
+      },
+      {
+        "description": "Reconstructed theta collision angle y value based on information from left station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_y_L"
+      },
+      {
+        "description": "Reconstructed theta collision angle y value based on information from right station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_y_R"
+      },
+      {
+        "description": "Reconstructed theta collision angle y value",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_y"
+      },
+      {
+        "description": "Reconstructed theta collision angle x value based on information from left station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_x_L"
+      },
+      {
+        "description": "Reconstructed theta collision angle x value based on information from right station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_x_R"
+      },
+      {
+        "description": "Reconstructed theta collision angle x value",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_x"
+      },
+      {
+        "description": "Reconstructed theta collision angle vector magnitude",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th"
+      },
+      {
+        "description": "Reconstructed four-momentum of the protons after collision",
+        "type": "double",
+        "unit": "GeV^2",
+        "variable": "t"
+      }
+    ],
+    "date_created": [
+      "2011"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "root",
+        "NTuple"
+      ],
+      "number_events": 12606699,
+      "number_files": 1,
+      "size": 440698145
+    },
+    "doi": "10.7483/OPENDATA.TOTEM.DZ6R.41RB",
+    "experiment": [
+      "TOTEM"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:a958f7e3",
+        "size": 440698145,
+        "uri": "root://eospublic.cern.ch//eos/opendata/totem/collision-data/2011_90m_7TeV/2011_RomanPot_PP/prepared_data.root"
+      }
+    ],
+    "keywords": [
+      "Roman Pot"
+    ],
+    "publisher": "CERN Open Data Portal",
+    "recid": "84000",
+    "title": "2011 Proton scattering data from Roman Pots at energy sqrt(s) = 7 TeV",
+    "type": {
+      "primary": "Dataset",
+      "secondary": [
+        "Derived"
+      ]
+    },
+    "usage": {
```

Comment:
I have put all the `use_with` part together with `usage` which is the default practice for dataset records.

Reply:
Fine with me